packages/utils/src/date/index.ts

```diff
@@ -175,10 +179,45 @@ export function timeZoneExists(timeZone: string): boolean {
 	}
 }
 
+/**
+ * Date format options
+ * @enum string
+ */
+export const FORMAT = {
+	/** en: June 29, 2021 / fr: 29 juin 2020 / ja: 2020年6月29日 */
+	MDY_LONG: 'MDY_LONG',
+	/** en: June 2020 / fr: juin 2020 / ja: 2020年6月 */
+	MY_LONG: 'MY_LONG',
+	/** en: 06/29/2020 / fr: 29/06/2020 / ja: 2020/06/29 */
+	MDY: 'MDY',
+	/** en: 6/29/20, 10:00 PM / fr: 29/06/2020 22:00 / ja: 2020/06/29 22:00 */
+	MDYHM: 'MDYHM',
+};
+
+const options: Map = {
+	[FORMAT.MDY_LONG]: { year: 'numeric', month: 'long', day: 'numeric' },
+	[FORMAT.MY_LONG]: { year: 'numeric', month: 'long' },
+	[FORMAT.MDY]: { year: 'numeric', month: '2-digit', day: '2-digit' },
+	[FORMAT.MDYHM]: { dateStyle: 'short', timeStyle: 'short' },
+};
+
+/**
+ * Format a date using Intl.
+ * @param date {DateFnsFormatInput} A date: Date, string or Number
+ * @param dateOption {string} Comes from FORMAT enum
+ * @param lang {string} language
+ * @returns The formated date
+ */
+export const format = (date: DateFnsFormatInput, dateOption: string, lang: string): string => {
```

Comment:
For consistency throughout the file:

```suggestion
export function format (date: DateFnsFormatInput, dateOption: string, lang: string): string {
```